.github/workflows/run-tests-with-beam.yaml

```diff
@@ -65,17 +65,17 @@ jobs:
           otp: "26"
           container: erlang:26
 
-        - os: "macos-latest"
-          otp: "23"
-          path_prefix: "/usr/local/opt/erlang@23/bin:"
-
         - os: "macos-latest"
           otp: "24"
           path_prefix: "/usr/local/opt/erlang@24/bin:"
 
         - os: "macos-latest"
           otp: "25"
           path_prefix: "/usr/local/opt/erlang@25/bin:"
+
+        - os: "macos-latest"
+          otp: "26"
+          path_prefix: "/usr/local/bin:"
```

Comment:
Maybe we want to use /usr/local/opt/erlang@26/bin as the path, as it will continue to work after OTP 27 is available in Homebrew.

Reply:
I thought about that, too, but unfortunately when you install `erlang@26`, it installs in `/opt/homebrew/Cellar/erlang`, i.e., it is unqualified, at least on my machine (macOS Sonoma 14.2.1).

```
> brew list | grep erlang
erlang@24

> ls -l /opt/homebrew/Cellar/erlang*
total 0
drwxr-xr-x  10 fadushin  admin  320 Jan 15 14:24 24.3.4.15/

> brew install erlang@26
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
netaddr                                 overtls                                 robin-map                               ssh3

You have 13 outdated formulae installed.

==> Downloading https://ghcr.io/v2/homebrew/core/erlang/manifests/26.2.2
==> Fetching erlang
==> Downloading https://ghcr.io/v2/homebrew/core/erlang/blobs/sha256:91596c2dda7291005c6b07c52f3df2d94371cbce4c3a86357813ecb21c93de97
==> Pouring erlang--26.2.2.arm64_sonoma.bottle.tar.gz
==> Caveats
Man pages can be found in:
  /opt/homebrew/opt/erlang/lib/erlang/man

Access them with `erl -man`, or add this directory to MANPATH.
==> Summary
🍺  /opt/homebrew/Cellar/erlang/26.2.2: 7,946 files, 493.4MB
==> Running `brew cleanup erlang`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

> brew list | grep erlang
erlang
erlang@24

> ls -l /opt/homebrew/Cellar/erlang*
/opt/homebrew/Cellar/erlang:
total 0
drwxr-xr-x  10 fadushin  admin  320 Feb 18 19:30 26.2.2/

/opt/homebrew/Cellar/erlang@24:
total 0
drwxr-xr-x  10 fadushin  admin  320 Jan 15 14:24 24.3.4.15/

```